### PR TITLE
feat: Add Chamber.IsOpen

### PR DESCRIPTION
**Pull request by Rysik5318**
## Description
-

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [x] I have checked the project can be compiled
- [x] I have tested my changes and it worked as expected

### CODE REVIEW

**Comment by Rysik5318** [comment]
Fuck

**Review by louis1706 — APPROVED** [review]
just do what i suggested

**Comment by Rysik5318** [comment]
@louis1706

**Comment by louis1706** [comment]
> @louis1706 
> 
> 

Yeah i didn't Believe you used this références are you sure it's work i would believe than it's client sided when using audio clip

**Comment by Rysik5318** [comment]
I checked, you're completely right

**Comment by Rysik5318** [comment]
Hmm, it still doesn't work, I'll have to do it.....

**Comment by Rysik5318** [comment]
That's the only way

**Comment by louis1706** [comment]
> That's the only way

Have you tested it ?

**Comment by Rysik5318** [comment]
> > That's the only way
> 
> Have you tested it ?

Yes. @louis1706

**Comment by louis1706** [comment]
> The only question I have is why exactly is in not in master… Like, lockers do need a property for being open on not, why wasnt it there in the first place

because Locker can have multiple chamber so you can't say it's open or not

File: EXILED/Exiled.API/Features/Lockers/Locker.cs [hunk 1]
```diff
@@ -91,6 +91,15 @@ public Locker(BaseLocker locker)
...
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the locker is currently open.
+        /// </summary>
+        public bool IsOpen
+        {
+            get => Chambers.First().IsOpen;
+            set => Chambers.First().IsOpen = value;
+        }
+
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/Lockers/Locker.cs:102]: probably revert this change (except if an other dev boy thinks it's good enough too keep)

**Comment by Rysik5318** [comment]
@louis1706

**Review by louis1706 — APPROVED** [review]
